tests/schemes/test_thrift.py

```diff
@@ -1162,6 +1163,6 @@ def test_headers_should_be_a_map_of_strings(headers):
 
     with pytest.raises(ValueError):
         yield tchannel.thrift(
-            request=True,
+            request=mock.MagicMock(),
```

Comment:
We still expect ValueError. Just because I change the order of the code so we use mock here.